src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/commands.py

```diff
@@ -47,7 +47,8 @@ def get_custom_sdk(custom_module, client_factory, resource_type=ResourceType.DAT
         g.command('delete', 'delete', confirmation=True)
         g.show_command('show', 'get_properties')
         g.custom_command('list', 'list_storage_accounts')
-        g.custom_command('show-usage', 'show_storage_account_usage')
+        g.custom_command('show-usage', 'show_storage_account_usage', min_api='2018-03-01-preview')
```

Comment:
when we say min_api, it is not greater than or equal to I believe. In that case shouldn't this be like 2018-02-01-preview

Reply:
@tjprescott, can you please clarify? I thought `min_api` means `>=`?

Reply:
Min API is greater than or equal to.

Reply:
So this change looks good to you?

Reply:
yes